be/osoc/common/views.py

```diff
@@ -521,3 +523,29 @@ class GithubLogin(SocialLoginView):
     adapter_class = GitHubOAuth2Adapter
     callback_url = "http://0.0.0.0:8000/accounts/github/login/callback/"
     client_class = OAuth2Client
+
+
+class CustomRegisterView(RegisterView):
+    """
+    This class overrides the registerview of rest_auth so users that get registered don't automatically
+    get logged in.
+    """
+    permission_classes = [permissions.IsAuthenticated, IsActive, IsAdmin]
+
+    def get_response_data(self, user):
+
+        if allauth_settings.EMAIL_VERIFICATION != \
+                allauth_settings.EmailVerificationMethod.NONE:
+            return {'detail': ('Verification e-mail sent.')}
+        return {'detail' : ('User has been created.')}
+
+    def perform_create(self, serializer):
+        user = serializer.save(self.request)
+        if allauth_settings.EMAIL_VERIFICATION != \
+                allauth_settings.EmailVerificationMethod.NONE:
+            complete_signup(
```

Comment:
do we want email verification in our tool?
since an admin creates an account for a user, i dont think this is necessary.
also, we dont want the newly created user to be logged in, but i think that the `complete_signup` function does this.
also, the allauth_settings are not recognized and are always none.

Reply:
yeah so:
1) I kept it in if we may later change it to use email verification, at the moment we should keep it I think.
2) The complete signup won't automatically login if the email verification is mandatory (which I will change) so that shouldn't be a problem.
3) This is the only one that i'm not so sure about why it does this, further testing is required.